jcl/src/java.base/share/classes/java/lang/invoke/MethodHandle.java

```diff
@@ -1236,9 +1236,11 @@ BoundMethodHandle rebind() {
 		throw OpenJDKCompileStub.OpenJDKCompileStubThrowError();
 	}
 	
+/*[IF Sidecar19-SE-OpenJ9]*/
```

Comment:
Does this have to be Java 9 only?  Given it's a compile stub, I'd be tempted to leave it in for Java 8 as well

Reply:
Its not used in Java 8 atm, but it doesn't need to be Java 9 only. Including it makes Java 8 code slightly bigger for no reason, but I can include it if you like.

Reply:
I'm fine with leaving it.  Its written now and is unlikely to need to be modified being compile stubs.